src/goose/brokers/redis/commands.clj

```diff
@@ -251,11 +251,33 @@
   (wcar* conn (car/zremrangebyscore sorted-set sorted-set-min score)))
 
 ;;; ============ Hashes ============
-
 (defn parse-map [conn hash]
   (wcar* conn (car/parse-map (car/hgetall hash) :keywordize)))
 
 ;;; ============ Misc ============
-
 (defn exists [conn key]
   (wcar* conn (car/exists key)))
+
+;;; ============ Batch ============
+(defn get-batch-state
```

Comment:
suggestion: After reading the name of the fn and `set-batch-state`, and the functionality of both, I think they will fit together better in the `brokers/redis/batch.clj` 
Dependencies look the same at first glance, wdyt

Reply:
Moved this fn to `brokers/redis/batch.clj`